### ISSUE TITLE
Sitewide: contrast + colour blindness testing

### ISSUE THREAD

**Issue by kimpham54** [dulibrarytech/digitaldu-frontend#65]
Priority from accessibility task force:

many tools to assess our site. then address any issues with this

**Comment by kimpham54** [comment]
from erin: https://www.color-blindness.com/coblis-color-blindness-simulator/

**Comment by kimpham54** [comment]
usability review should be done on this site for 2025 - close this one for now
